404.html

```diff
@@ -7,10 +7,16 @@
     {
         "imports": {
             "@dropins/tools/": "/scripts/__dropins__/tools/",
-            "@dropins/storefront-pdp/": "/scripts/__dropins__/storefront-pdp/"
+            "@dropins/storefront-cart/": "/scripts/__dropins__/storefront-cart/",
+            "@dropins/storefront-checkout/": "/scripts/__dropins__/storefront-checkout/",
+            "@dropins/storefront-pdp/": "/scripts/__dropins__/storefront-pdp/",
+            "@dropins/storefront-order-confirmation/": "/scripts/__dropins__/storefront-order-confirmation/",
+            "@dropins/storefront-account/": "/scripts/__dropins__/storefront-account/",
+            "@dropins/storefront-order/": "/scripts/__dropins__/storefront-order/",
+            "@dropins/storefront-auth/": "/scripts/__dropins__/storefront-auth/"
```

Comment:
```suggestion
            "@dropins/storefront-cart/": "/scripts/__dropins__/storefront-cart/",
            "@dropins/storefront-auth/": "/scripts/__dropins__/storefront-auth/"
```

Should only need things which are used in nav bar.